Makefile

```diff
@@ -169,6 +171,31 @@ else
             $(TESTS)
 endif
 
+jstest: TESTS ?= UI/tests
+jstest:
```

Comment:
If the tests get integrated into prove, you can trigger the required subset with arguments to the price invocation.